frontend/src/components/ui/multi-select.tsx

```
@@ -0,0 +1,269 @@
+              </div>
+
+              {filteredSuggestions.length > 0 && (
+                <>
+                  {/* Select All option */}
+                  <div
+                    className="flex items-center gap-2 px-3 py-2 hover:bg-accent cursor-pointer sticky top-[52px] bg-popover border-b border-border z-10"
+                    onClick={(e) => {
+                      e.stopPropagation();
+                      handleSelectAll();
+                    }}
+                  >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)